server/common/Configuration.java

```diff
@@ -140,6 +142,126 @@ public VaticleFactory vaticleFactory() {
         return vaticleFactory;
     }
 
+    public static class Server {
+
+        private static final String name = "server";
+        private static final String description = "Server and networking configuration.";
+
+        private final InetSocketAddress address;
+
+        private Server(InetSocketAddress address) {
+            this.address = address;
+        }
+
+        static class Parser extends ValueParser.Nested<Server> {
+
+            private static final EntryParser<InetSocketAddress> addressParser = TypedValue.create("address", "Address to listen for GRPC clients on.", INET_SOCKET_ADDRESS);
```

Comment:
the `Server` in Core only requires `address` as an entry, which must be an `InetSocketAddress`